libs.jaxb/external/jaxb-2.2.5-2-xjc-license.txt

```diff
@@ -0,0 +1,382 @@
+Name: JAXB
+Description: JAXB Reference Implementation
+Version: 2.2.5-2
+Origin: Oracle
+URL: https://jaxb.java.net/
+License: CDDL-1.1-MIT-jaxb-xjc
+Files: jaxb-xjc-2.2.5-2.jar
+
+Parts of the work are:
```

Comment:
Same here. I assume it's a fat JAR, with the org/kohsuke package under one license, com/sun under another, etc?